tests/integration-tests/conftest.py

```diff
@@ -179,6 +179,11 @@ def pytest_addoption(parser):
         help="Name of CFN stack providing database stack to be used for testing Slurm accounting feature.",
     )
     parser.addoption("--external-shared-storage-stack-name", help="Name of existing external shared storage stack.")
+    parser.addoption(
+        "--force-run-instance-fleet-manager",
+        help="Force usage of Run-instance-api instead of creat-fleet api for compute fleet",
```

Comment:
Force usage of run-instances boto3 API instead of create-fleet for compute fleet scaling up. Note: if there are multiple instances in the list, only the first will be used.

Reply:
Will add this